pkg/config/signer_test.go

```diff
@@ -106,52 +108,43 @@ func TestNewSigner(t *testing.T) {
 }
 
 func TestECDSAPublicKeyImport(t *testing.T) {
-	t.Run("certificate does not contain valid ecdsa publicKey", func(t *testing.T) {
-		gt := NewGomegaWithT(t)
-		x509cert := &x509.Certificate{PublicKey: struct{}{}}
-		_, err := ecdsaPublicKeyImport(x509cert)
-		gt.Expect(err).To(MatchError("certificate does not contain valid ECDSA public key"))
-	})
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+	x509cert := &x509.Certificate{PublicKey: struct{}{}}
+	_, err := ecdsaPublicKeyImport(x509cert)
+	gt.Expect(err).To(MatchError("certificate does not contain valid ECDSA public key"))
 }
 
 func TestECDSAPrivateKeyImport(t *testing.T) {
-	t.Run("nil private key", func(t *testing.T) {
-		gt := NewGomegaWithT(t)
-		_, err := ecdsaPrivateKeyImport(nil)
-		gt.Expect(err.Error()).To(ContainSubstring("invalid key type. The DER must contain an ecdsa.PrivateKey"))
-	})
+	t.Parallel()
+
+	gt := NewGomegaWithT(t)
+	_, err := ecdsaPrivateKeyImport(nil)
+	gt.Expect(err.Error()).To(ContainSubstring("invalid key type. The DER must contain an ecdsa.PrivateKey"))
```

Comment:
I much more prefer matching the actual error as opposed to just checking for a substring

Reply:
in this case, and previous ones, the error message contains bytes of data that causes the error, such as _invalid private key_ bytes. Therefore, we decided not to match the exact error message.